global-configuration/src/main/resources/archetype-resources/src/main/resources/index.jelly

```diff
@@ -0,0 +1,4 @@
+<?jelly escape-by-default='true'?>
+<div>
+    TODO
```

Comment:
Intentionally empty?

Reply:
Well…for the `empty-plugin` archetype it is intentionally empty—a placeholder, reminder that you need to fill in something here. For archetypes which contain actual content, and thus which serve more as examples than actual starting points for new plugins (I guess?), it is not so obvious. But from the flip side: if there _were_ a real description here, who would it serve? Not useful during exploration (`hpi:run` etc.), only to actual users of a plugin released on the UC—in which case the author / archetype instantiator had better be editing this.

Reply:
Yeah I didn't have a problem with it, I just wanted to double check.